hibernate-core/src/main/java/org/hibernate/stat/Statistics.java

```diff
@@ -15,6 +15,8 @@
  */
 public interface Statistics {
 
+	int DEFAULT_QUERY_STATISTICS_MAX_SIZE = 5000;
+
```

Comment:
I think I would have put the default in `AvailableSettings` with the others.

Reply:
The `AvailableSettings` is used only for the global configuration property names. I chose the `Statistics` interface since this default value is only relevant for statistics.